rabbit_monitor.py

```diff
@@ -38,13 +41,16 @@
 healthcheck_url = settings.rabbit_url + 'healthchecks/node'
 aliveness_url = (settings.rabbit_url +
                  'aliveness-test/{}'.format(settings.rabbit_default_vhost))
+
 message_url = (settings.rabbit_url +
                'overview/')
 
+nodes_url = settings.rabbit_url + 'nodes'
+
 urls = {'healthcheck': healthcheck_url,
         'aliveness': aliveness_url,
         'messages': message_url,
-        }
+        'nodes': nodes_url}
```

Comment:
A bit pedantic, but can you format this as:

```python
urls = {'healthcheck': healthcheck_url,
         'aliveness': aliveness_url,
         'messages': message_url,
         'nodes': nodes_url,
       }
```

The trailing comma means that when we add to the `Dict` we don't get git revision on both lines, which makes it easier to review.